tfhe/src/integer/server_key/radix_parallel/tests_signed/modulus_switch_compression.rs

```diff
@@ -1,15 +1,13 @@
 use crate::integer::gen_keys_radix;
-use crate::integer::tests::create_parameterized_test_classical_params;
+use crate::integer::tests::create_parameterized_test;
 #[cfg(tarpaulin)]
 use crate::shortint::parameters::coverage_parameters::*;
 use crate::shortint::parameters::*;
 use rand::Rng;
 
 const NB_TESTS: usize = 10;
 
-// Remove multi bit PBS parameters as
-// modulus switch compression and multi bit PBS are currently not compatible
-create_parameterized_test_classical_params!(modulus_switch_compression_signed);
+create_parameterized_test!(modulus_switch_compression_signed);
```

Comment:
are we sure all parameter sets in the parameterized test will be compatible properly for compression ?

Reply:
Should be working as parameters have `message_modulus=carry_modulus`

Reply:
ah it's not the full compression, it's the "old one"

Reply:
Yes, I noticed it was also not tested with multi params even if it is supported